### PR TITLE
Tracking Branch for the Ember Website Redesign RFC

### PR DESCRIPTION
**Pull request by mansona**
This PR is intended to track the progress of the Ember Website Redesign. This redesign has had an RFC process and the full content of the RFC can be seen here: https://emberjs.github.io/rfcs/0425-website-redesign.html

If you would like to follow this process take a look at the [Website Redesign Github Project](https://github.com/orgs/ember-learn/projects/16)

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
Is there anyway to release this incrementally, maybe per-page? Rather than all at once?

**Comment by mansona** [comment]
@NullVoxPopuli I completely missed your comment (sorry 🙈). Unfortunately no, there is no way to  release this "page by page" and even if we could it would give a very bad experience overall 😞 

There have been some discussions about releasing the Website (emberjs.com) before each of the other projects (blog, guides, api docs, deprecations app) are done but it's up to the Framework team to decide. We are almost ready to release the Website (just emberjs.com) and we will see what is decided about how it is going to be released 👍

File: app/templates/index.hbs [hunk 1]
```diff
@@ -1,123 +1,277 @@
...
+    </div>
+    <div class="layout mt-4">
+      <div class="lg:col-4 lg:start-3">
+        <img class="img-fluid" src="/images/home/ember-performance.png" alt="" role="presentation" width="636" height="424"
+        srcset="/images/home/ember-performance-2x.png 2x">
+      </div>
+      <div class="lg:col-2">
+        <h3 class="text-lg">Performance</h3>
+        <p>
+          Ember is built on the Glimmer rendering engine, which compiles templates down to a highly performant DOM Virtual Machine. The Glimmer complier builds specific "programs" that are designed to create or update your node directly in the DOM without the need for a Virtual DOM, making it one of the fastest rendering technologies on the market today.
```
Comment (knownasilya) [inline app/templates/index.hbs]:
```suggestion
          Ember is built on the Glimmer rendering engine, which compiles templates down to a highly performant DOM Virtual Machine. The Glimmer compiler builds specific "programs" that are designed to create or update your node directly in the DOM without the need for a Virtual DOM, making it one of the fastest rendering technologies on the market today.
```

**Comment by knownasilya** [comment]
The dropdown menus in the navbar go under the leaflet map on the meetups page.

File: app/templates/index.hbs (hunk 1 above)
Comment (mansona) [reply app/templates/index.hbs]: thanks! this is already fixed in https://github.com/ember-learn/ember-website/pull/560 👍

**Comment by NullVoxPopuli** [comment]
Omg. Yay :haha:

**Pull request merged**